### ISSUE TITLE
Tiled gnome-terminal windows have margins on right and bottom sides even though set to zero

### ISSUE THREAD

**Issue by hermidalc** [gTile/gTile#86]
On Gnome 3.32 when you tile gnome-terminal windows they have margins on the right and bottom sides even though gTile margins are set to zero.  Would expect that their sides are flush with each other and with edges of the workspace

**Comment by scherepanov** [comment]
Well, I would expect it too.
It is a feature of gnome-terminal. It calculates required size of window based on font width and height, and force terminal window size to adjust.
I kinda understand gnome-terminal developers why it is done.
On other side, if you force gnome-window resize using gnome built-in, like left/right half of screen, or maximize vertically/horizontally/full screen, it follows directions and do what it was asked to.

In short, it is not gTile fault, it is a feature of gnome-terminal.

**Comment by hermidalc** [comment]
Sorry just want some clarification... when I use gTile to tile gnome-terminal windows and it leaves a margin on the bottom and right sides I can then manually correct it with my mouse by moving the edge one grid step on the invisible terminal desktop grid and they all become flush. So if I can set them to all be flush without resorting to the gnome-window resize built-in why is it not possble for gTile to make them flush?

**Comment by scherepanov** [comment]
I checked
From what I can see, gnome-terminal windows are not really flush. There is always some overlap. Probably in your case it is as small as one pixel, and it is hard to recognize on a screen.
gTile uses Gnome move_resize_frame, and provide upper left corner and width/height. All those are calculated so windows are flush. It does works properly, as you can see on other applications.

**Comment by hermidalc** [comment]
On my Fedora 30 they are flush except as you said maybe one pixel overlap which you cannot see.

As an experiment, with your gTile dev env what if you tried to do GNOME move_resize_frame() on gnome-terminal windows and calculate the width and height to be slightly larger than it expected to be flush?  What happens?
